### PR TITLE
fixing faenet issue

### PR DESCRIPTION
**Pull request by ginihumer**
faenet issue regarding the prediction of very large energies

adding batch run script 

basic logger

### CODE REVIEW

File: activelearning/dataset/ocp.py [hunk 1]
```diff
@@ -25,6 +26,15 @@ def __init__(
         self.appended_data = []
         self.return_target = return_target
 
+        # https://github.com/RolnickLab/ocp/blob/c93899a23947cb7c1e1409cf6d7d7d8b31430bdd/ocpmodels/trainers/base_trainer.py#L361
+        # https://github.com/RolnickLab/ocp/blob/c93899a23947cb7c1e1409cf6d7d7d8b31430bdd/configs/models/tasks/is2re.yaml#L25
+        # https://github.com/RolnickLab/ocp/blob/c93899a23947cb7c1e1409cf6d7d7d8b31430bdd/ocpmodels/modules/normalizer.py#L11
+        self.target_normalizer = Normalizer(
+            mean=-1.525913953781128,
+            std=2.279365062713623,
```
Comment (alexhernandezgarcia) [inline activelearning/dataset/ocp.py]: I would add these numbers as "global variables" in the script, together with the references.

File: activelearning/dataset/ocp.py [hunk 2]
```diff
@@ -132,7 +161,8 @@ def __init__(
             )
 
     def maxY(self):
-        return 10  # -> TODO: what are the actual bounds?
+        # return 10  # -> TODO: what are the actual bounds?
+        return 5  # when target is standard normalized
```
Comment (alexhernandezgarcia) [inline activelearning/dataset/ocp.py:181]: How come the min and max are the same?

**Review by alexhernandezgarcia — APPROVED** [review]
Looks good! Does this solve the issues? Please see my comments before merging.

File: activelearning/dataset/ocp.py (hunk 2 above)
Comment (ginihumer) [reply activelearning/dataset/ocp.py:181]: it's normalized

File: activelearning/dataset/ocp.py (hunk 1 above)
Comment (ginihumer) [reply activelearning/dataset/ocp.py]: updated

File: activelearning/dataset/ocp.py (hunk 2 above)
Comment (alexhernandezgarcia) [reply activelearning/dataset/ocp.py:181]: I still don't get it, but ok I guess it doesn't matter.

**Pull request merged**